tools/nsenter-node.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+# FROM https://github.com/alexei-led/nsenter/blob/master/nsenter-node.sh
+# USAGE: ./nsenter-node.sh node-name
+set -x
```

Comment:
@phudtran Please add MIT license

Reply:
It's from another repo under GPL. I'll just remove it for now